library/src/main/java/com/cube/storm/ui/fragment/StormBottomTabsFragment.java

```diff
@@ -185,19 +184,20 @@ public boolean onTabSelected(int position, boolean wasSelected)
 		return true;
 	}
 
+	// When the bottomNavigation layout change, set the tab item content descriptions for every tab
+	@Override public void onLayoutChange(View v, int left, int top, int right, int bottom, int oldLeft, int oldTop, int oldRight, int oldBottom)
+	{
+		setTabItemContentDescriptions();
+	}
+
 	/**
 	 * This method set the content descriptions for all items from the bottom navigation menu.
 	 * Check if the current tab is selected to update the content description comment
 	 */
 	private void setTabItemContentDescriptions()
 	{
-		if (viewPager.getAdapter() == null)
-		{
-			return;
-		}
 		int tabCount = bottomNavigation.getItemsCount();
-		int maxElements = Math.min(viewPager.getAdapter().getCount(), MAX_BOTTOM_TABS);
-		for (int bottomTabIdx = 0; bottomTabIdx < maxElements; bottomTabIdx++)
+		for (int bottomTabIdx = 0; bottomTabIdx < tabCount; bottomTabIdx++)
```

Comment:
Will that create issues with the max size of navigation bar being 5?? Cos not the number can be even bigger.

Reply:
The bottom navigation button always will be smaller or equals than 5, so it is safe gets the _tabCount_ variable to checks the number of tabs available.

Reply:
I can not see that on the codebase... i do not know how that will be the case.

Reply:
Hi Nikos, as we have discussed before, _max_tab_ param is not necessary because the AHBottomNavigationBar has a static param (_MAX_ITEMS_) which control the maximum number of tabs. So, the Bottom Navigation Bar always gets five or fewer elements.